packages/core/src/App/Containers/WarningScamMessageModal/warning-scam-message.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { connect } from 'Stores/connect';
+import { Modal, DesktopWrapper, MobileWrapper, Button, Icon } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { PropTypes } from 'prop-types';
+import { withRouter } from 'react-router';
+import './warning-scam-message-modal.scss';
+import WarningScamMessageModal from './warning-scam-message-modal';
+import WarningScamMessageTitle from './warning-scam-message-title';
+import WarningScamMessageContent from './warning-scam-message-content';
+import WarningScamMessageCheckbox from './warning-scam-message-checkbox-content';
+
+const WarningScamMessage = ({ is_logged_in, client_country }) => {
+    const [read_message, setReadMessage] = React.useState(false);
```

Comment:
I believe we should name Boolean variables with 'is' prefix in name, let say: is_message_read.
it is more explicit , how do you think?

Reply:
sounds better will rename it according to your suggestion 😄